src/components/user/dialogs/ConnectionsDialog.tsx

```diff
@@ -14,6 +14,8 @@ import {
   useGetFollowing,
 } from '@/hooks/user/useUserConnections';
 
+import { FollowButton, FollowButtonTab } from '../buttons/followButton';
```

Comment:
FollowButton and FollowButtonTab are identical components, what's the difference?

Reply:
i had to change the margin and padding of the unfollow button as asked by piyush but if I did that it would change the unfollow button everywhere so I made a new button with different padding and imported it. to display int he follwoing tab.

Reply:
We are using tailwind merge, for overriding classes. Why not use that instead?